acceptance/tests-v2/mesh/mesh_inject_test.go

```diff
@@ -0,0 +1,79 @@
+package mesh
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/hashicorp/consul-k8s/acceptance/framework/connhelper"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/consul"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/helpers"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/k8s"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/logger"
+)
+
+// Test that Connect works for an application with multiple ports. The multiport application is a Pod listening on
+// two ports. This tests inbound connections to each port of the multiport app, and outbound connections from the
+// multiport app to static-server.
+func TestMeshInject_MultiportService(t *testing.T) {
+	for _, secure := range []bool{false} {
+		name := fmt.Sprintf("secure: %t", secure)
+
+		t.Run(name, func(t *testing.T) {
+			cfg := suite.Config()
+			cfg.SkipWhenOpenshiftAndCNI(t)
```

Comment:
Since this test also only works for tproxy, should we skip if the tproxy flag isn't enabled on the CLI? That way CI won't try to run it twice.

Reply:
Yes! great catch thanks!